Libplanet/Blockchain/Policies/BlockPolicy.cs

```diff
@@ -13,6 +14,8 @@ namespace Libplanet.Blockchain.Policies
     public class BlockPolicy<T> : IBlockPolicy<T>
         where T : IAction, new()
     {
+        private readonly Predicate<Transaction<T>> _isTransactionValid;
```

Comment:
`IsTransactionInvalid` method just calls `_isTransactionsValid` and pass its return value. so how about using property? 

```suggestion
        public Predicate<Transaction<T>> IsTransactionValid { get; }
```

Reply:
@moreal I believe the way you suggested does not satisfy the interface anyway, because in the IL level delegate properties and methods are merely distinct each other; the same looking syntax between delegate properties and methods in C# is just a coincidence.

Reply:
@dahlia If I had understood correctly, it means that method `bool IsTransactionValid(Transaction<T>)` and delegate property `Predicate<Transaction<T>>` are distinct each other so it will not be achieved.

If then, it was what I really wanted to suggest to replace `IBlockPolicy><T>.IsTransactionValid(Transaction<T>)` with `Predicate<Transaction<T>>`. As code, it will be like below.

```csharp
public interface IBlockPolicy<T>
    where T : IAction, new()
{
    // bool IsTransactionValid(Transaction<T> transaction);
    Predicate<Transaction<T>> IsTransactionValid { get; }
}
```

Reply:
@moreal That was how this patch looked before it's amended, but then it became like the current way in a reason: .NET interfaces say nothing about constructors, so an interface anyway cannot enforce its implementations to take a delegate. On the other way, it's unclear what's benefit of having a predicate as a delegate property over a method, which is more common and natural.

Reply:
Okay, I understood. Thanks for your good explanation.